### PR TITLE
Add HMS floor plan export

### PR DESCRIPTION
**Pull request by shaded-enmity**
This PR adds data export to the HMS bucket for analytics.

RHINENG-9418

/ping @vkrizan @skateman

### CODE REVIEW

File: clowdapp.yaml
Comment (vkrizan) [inline clowdapp.yaml:201]: @shaded-enmity do you have experience with having JSON formatted data in a Parquet format? I do not seem to find it in the [supported Parquet types](https://parquet.apache.org/docs/file-format/types/).

**Review by vkrizan — APPROVED** [review]
Thanks!

Waiting on the deployment changes before the merge.

File: clowdapp.yaml
Comment (shaded-enmity) [reply clowdapp.yaml:201]: Here's how it works in Image Builder, https://github.com/osbuild/image-builder/blob/2139543853a11e41577664d36e39422c618bac4b/templates/image-builder.yml#L158
packages + repositories is a list of objects. I think it's handled under the hood via the Logical Types, but I haven't looked at how exactly this is implemented.

**Pull request merged**
